CodeEdit/Features/WindowCommands/MainCommands.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MainCommands.swift
+//  CodeEdit
+//
+//  Created by Wouter Hennen on 13/03/2023.
+//
+
+import SwiftUI
+import Sparkle
+
+struct MainCommands: Commands {
+
+    @Environment(\.openWindow) var openWindow
+
+    var body: some Commands {
+        CommandGroup(replacing: .appInfo) {
+            Button("About CodeEdit") {
+                openWindow(id: "About")
+            }
+
+            Button("Check for updates...") {
+                NSApp.sendAction(#selector(SPUStandardUpdaterController.checkForUpdates(_:)), to: nil, from: nil)
+            }
+        }
+
+        CommandGroup(after: .appSettings) {
```

Comment:
Why not swizzle the Settings menu item like the Open Recents one and open the actual settings like that instead of having this hidden one?